t/14-isotovideo.t

```diff
@@ -128,49 +128,60 @@ subtest 'isotovideo with git refspec specified' => sub {
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is($repos[0], 'https://github.com/foo/bar.git', 'repo with full URL');
     is(scalar @repos, 1, 'one wheel');
     $specfile->spurt("version: v0.1\nwheels: [https://github.com/foo/bar.git#branch]");
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is($repos[1], 'https://github.com/foo/bar.git#branch', 'repo URL with branch');
     is(scalar @repos, 2, 'one wheel');
     $specfile->spurt("version: v0.1\nwheels: [foo/bar]");
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is(scalar @repos, 3, 'one wheel');
     is($repos[2], 'https://github.com/foo/bar.git', 'only wheel');
     $specfile->spurt("version: v0.1\nwheels: [foo/bar, spam/eggs]");
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is($repos[4], 'https://github.com/spam/eggs.git', 'second wheel');
     is(scalar @repos, 5, 'two wheels');
     $specfile->remove;
-    is(checkout_wheels("$wheels_dir"), 1, 'no wheels');
+    is(checkout_wheels($case_dir, $wheels_dir), 1, 'no wheels');
     is(scalar @repos, 5, 'git never called');
 
     # also verify that isotovideo invokes the wheel code correctly
-    $bmwqemu::vars{CASEDIR} = "$data_dir/tests";
     $specfile->spurt("version: v0.1\nwheels: [copy/writer]");
-    path($pool_dir, 'writer', 'lib', 'Copy', 'Writer')->make_path->child('Content.pm')->spurt("package Copy::Writer::Content; use Mojo::Base 'Exporter'; our \@EXPORT_OK = qw(write); sub write {}; 1");
-    path($pool_dir, 'writer', 'tests', 'pen')->make_path->child('ink.pm')->spurt("use Mojo::Base 'basetest'; use Copy::Writer::Content 'write'; sub run {}; 1");
-    my $log = combined_from { isotovideo(
-            opts => "wheels_dir=$wheels_dir casedir=$data_dir/tests schedule=pen/ink _exit_after_schedule=1") };
-    like $log, qr@Skipping to clone.+copy/writer@, 'already cloned wheel picked up';
-    like $log, qr/scheduling ink/, 'module from the wheel scheduled';
-    rmtree "$pool_dir/writer";
+
+    for my $dir (("$data_dir/wheels_dir", undef)) {
+        chdir($pool_dir);
+        unlink('vars.json') if -e 'vars.json';
+        $wheels_dir = $dir // $pool_dir;
+        $bmwqemu::vars{CASEDIR} = $case_dir;
+        $bmwqemu::vars{SCHEDULE} = 'pen/ink';
+        $bmwqemu::vars{WHEELS_DIR} = $dir;
+
+        subtest "wheels in $wheels_dir" => sub {
+            path($wheels_dir, 'writer', 'lib', 'Copy', 'Writer')->make_path->child('Content.pm')->spurt("package Copy::Writer::Content; use Mojo::Base 'Exporter'; our \@EXPORT_OK = qw(write); sub write {}; 1");
+            path($wheels_dir, 'writer', 'tests', 'pen')->make_path->child('ink.pm')->spurt("use Mojo::Base 'basetest'; use Copy::Writer::Content 'write'; sub run {}; 1");
+            checkout_wheels($case_dir, $wheels_dir);
+            my $log = lives_ok { combined_from { load_test_schedule } } 'no error';
```

Comment:
`lives_ok` does not return a log

Reply:
This was meant to keep the test plan from breaking when git fails. I guess we don't need it with your approach. The non-working case is finally less annoying and fails instantly!